### PR TITLE
Added MMIX Example

### PR DESCRIPTION
**Pull request by yamamushi**

### CODE REVIEW

File: examples/mmix_rockstar.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from RockStar import RockStar
```
Comment (avinassh) [inline examples/mmix_rockstar.py]: should be:

```
 from rockstar import RockStar
```

**Comment by yamamushi** [comment]
Updated, but please let me know if you want me to squash the commits :)

**Comment by avinassh** [comment]
Yup, please do.

**Comment by yamamushi** [comment]
Squashed :)

**Comment by avinassh** [comment]
Merged, thanks!

**Pull request merged**